.github/workflows/coveralls.yml

```
@@ -0,0 +1,27 @@
+name: Coveralls
+
+on: ["push", "pull_request"]
```

We should only do this for PRs and pushes to master. That's also how it has been done in Travis.

```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

---

Done